### DISCUSSION TITLE
更新最新版本后，无法登陆

### DISCUSSION THREAD

**Discussion by d15801543974** [community/community#17659] [Copilot Conversations] [answered]
IntelliJ IDEA 2021.2.2，更新插件后一直卡在 Waiting for GitHub Authentication。
不使用代理和使用代理都试过，都是失败的。GitHub Copilot  1.1.24.1610

**Comment by d15801543974** [comment]
我在上一个版本中还可以正常使用。

**Reply by TDPHY** [reply]
我也是,我上一个版本还可以的,这个版本不行,然后我就回滚到上一个版本,结果上一个版本都不行了

**Reply by d15801543974** [reply]
> 我也是，我的上一个版本还可以的，这个版本不行，然后我就回滚到上一个版本，结果上一个版本不能行了

测试版就是不靠谱呀！

**Reply by JunQu** [reply]
相同的问题，但是我觉得还是去英文下面问吧，可以帮助到更多的人，也有管理员回复，直接在那个问题下添加吧

https://github.com/github/feedback/discussions/17617

**Reply by zchancy** [reply]
我上个版本不行 今天更新就行了

**Comment by d15801543974** [comment]
![image](https://user-images.githubusercontent.com/23231997/171585993-3b79387a-5c7c-4e2c-b80e-6d242c92090c.png)

**Comment by d15801543974** [comment]
我还是挂了代理才能到 https://github.com/login/device 这个有页面的

**Comment by Zer0ON1** [comment]
1.1.16可以，我的是1.1.20.x之后唤起登陆都不行了

**Comment by d15801543974** [comment]
我的现在可以了，就是一直尝试。说不定哪次就可以了。随缘吧

**Comment by ettaboyle** [comment, marked as answer]
Thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. What this means is that, until that changes, we need to ask that everyone use English here in the GitHub Community Forum when posting. 

Please translate whenever possible and default to English when posting. We are actively working to better support our global users and more languages, but this will help you get a response faster, just as @JunQu mentioned in their comment above :)
